docs/write_jupyter_notebook.md

```diff
@@ -0,0 +1,35 @@
+# Example 9: Use dbt from a Jupyter Notebook
+
+We also offer fal as an importable package to load in a Python environment to reference and use your dbt tables in a dynamic way.
+
```

Comment:
maybe add a sentence here how you might want to import fal to a jupiter notebook to run some ad-hoc analysis